socrata/sources.py

```diff
@@ -256,7 +256,7 @@ def add_to_revision(self, uri, revision):
         """
         Associate this Source with the given revision.
         """
-        return self._mutate(patch(
+        return self._clone(patch(
             self.path(uri),
```

Comment:
Associating a source with a revision is just a join table entry, right? Does it matter if its clone/mutate?

Reply:
dsmapi handles the re-association when cloning